core/base/inc/TString.h

```diff
@@ -436,7 +451,7 @@ friend Bool_t  operator==(const TString &s1, const char *s2);
 #endif
    ;
 
-   ClassDef(TString,2)  //Basic string class
+   ClassDef(TString,3)  //Basic string class
```

Comment:
Only needed if you change the on-disk representation of `TString` - which you didn't. So please keep version `2`.